### ISSUE TITLE
"Vacuum.sc" script compatibility 

### ISSUE THREAD

**Issue by chris819t** [gnembon/scarpet#271]
**Is your feature request related to a problem? Please describe.**
Sort of. When I crouch and mine a block (a modified version of your script) with a vacuum shulker that contains the blocks that are in the shulkerbox it doesnt go into the appropriate shulkerbox but instead go straight into my inventory instead of going into the "vacuum" shulkerbox

**Describe the solution you'd like**
When having a shulkerbox of at least a single slot and item of deepslate inside of a "vacuum" shulkerbox the deepslate should go into said shulkerbox instead of going into your normal inventory 

**Describe alternatives you've considered**
Sorry, if I knew how to code I would provide some suggestions to make that work.

**Additional context**
N/A

**Comment by Firigion** [comment]
I think this should be solvable quite elegantly with a custom event.

**Comment by chris819t** [comment]
custom event? Do you mean by editing the code slightly? I'm not really smart enough to do so sadly :|

**Comment by Firigion** [comment]
I wasn't proposing you do it, don't worry. It ewas more a note for whoever decides to tackle this report. Even maybe myself at some point in the future.

**Comment by chris819t** [comment]
Ah ok. I hope you can at some point soon in time because I know its possible since it was a thing in 1.16.5 with careful break

**Comment by Ghoulboy78** [comment]
Yep, that's something I wanted to tackle in #307 too, but wasn't sure where to start. But I am planning on fixing this issue there
